python/py-virtualenv/Portfile

```diff
@@ -46,7 +66,7 @@ if {${name} ne ${subport}} {
     post-destroot {
         set docdir ${prefix}/share/doc/${subport}
         xinstall -d ${destroot}${docdir}/docs/changelog
-        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE.txt AUTHORS.txt \
+        xinstall -m 0644 -W ${worksrcpath} README.md LICENSE \
```

Comment:
is this file needed? why?

Reply:
In a previous review @reneeotten made me keep these files. :)